base/base_tests/stl_helpers_test.cpp

```diff
@@ -61,9 +61,9 @@ template <template <typename...> class Cont>
 void TestEqualsBy()
 {
   {
-    using Value = pair<int, int>;
+    using Value = std::pair<int, int>;
```

Comment:
Why not to just put "using namespace std" at the top of source files? The number of changes will be significantly smaller.